plugins/maven/README.md

```diff
@@ -56,31 +56,33 @@ This Maven plugin is able to generate the following classes based on your GraphQ
-| `requestSuffix`                        | String             | Request                                   | Sets the suffix for `Request` classes. |
-| `responseProjectionSuffix`             | String             | ResponseProjection                        | Sets the suffix for `ResponseProjection` classes. |
+| Option                                          | Data Type          | Default value                             | Description |
+| :---------------------------------------------: | :----------------: | :---------------------------------------: | ----------- |
+| `graphqlSchemaPaths`                            | List(String)       | (falls back to `graphqlSchemas`)          | GraphQL schema locations. You can supply multiple paths to GraphQL schemas. To include many schemas from a folder hierarchy, use the `graphqlSchemas` block instead. |
+| `graphqlSchemas`                                | *See table below*  | All `.graphqls` files in resources        | Block to define the input GraphQL schemas, when exact paths are too cumbersome. See table below for a list of options. |
+| `outputDir`                                     | String             | None                                      | The output target directory into which code will be generated. |
+| `jsonConfigurationFile`                         | String             | Empty                                     | Path to an external mapping configuration. |
+| `packageName`                                   | String             | Empty                                     | Java package for generated classes. |
+| `apiPackage`                                    | String             | Empty                                     | Java package for generated api classes (Query, Mutation, Subscription). |
+| `modelPackage`                                  | String             | Empty                                     | Java package for generated model classes (type, input, interface, enum, union). |
+| `generateBuilder`                               | Boolean            | True                                      | Specifies whether generated model classes should have builder. |
+| `generateApis`                                  | Boolean            | True                                      | Specifies whether api classes should be generated as well as model classes. |
+| `generateAsyncApi`                              | Boolean            | False                                     | If true, then wrap type into `java.util.concurrent.CompletableFuture` or `subscriptionReturnType` |
+| `generateDataFetchingEnvironmentArgumentInApis` | Boolean            | False                                     | If true, then `graphql.schema.DataFetchingEnvironment env` will be added as a last argument to all methods of root type resolvers and field resolvers. |
```

Comment:
Sorry for coming up with such a long name, I tried removing some of the words but it really feels like they are all needed 😅

Reply:
Haha, true 🤣 
Spring-style!